packages/cli/src/databases/entities/test-run.ee.ts

```diff
@@ -35,4 +35,17 @@ export class TestRun extends WithTimestampsAndStringId {
 
 	@Column(jsonColumnType, { nullable: true })
 	metrics: AggregatedTestRunMetrics;
+
+	// Total number of the test cases, matching the filter condition of the test definition (specified annotationTag)
+	@Column('integer', { nullable: true })
+	total: number;
+
+	// Number of test cases that passed (evaluation workflow was executed successfully)
+	@Column('integer', { nullable: true })
+	passed: number;
```

Comment:
Would it make sense to have a `num` prefix for all these to show they are numeric columns? I.e. `numTotalCases`, `numPassedCases`, `numFailedCases`.